src/codemagic/apple/resources/enums.py

```diff
@@ -78,30 +79,73 @@ class CapabilitySettingKey(ResourceEnum):
 
 
 class CapabilityType(ResourceEnum):
+    """
+    https://developer.apple.com/documentation/appstoreconnectapi/capabilitytype
+    """
+
     ACCESS_WIFI_INFORMATION = "ACCESS_WIFI_INFORMATION"
-    APP_GROUPS = "APP_GROUPS"
+    APPLE_ID_AUTH = "APPLE_ID_AUTH"
     APPLE_PAY = "APPLE_PAY"
+    APP_GROUPS = "APP_GROUPS"
     ASSOCIATED_DOMAINS = "ASSOCIATED_DOMAINS"
     AUTOFILL_CREDENTIAL_PROVIDER = "AUTOFILL_CREDENTIAL_PROVIDER"
     CLASSKIT = "CLASSKIT"
+    COREMEDIA_HLS_LOW_LATENCY = "COREMEDIA_HLS_LOW_LATENCY"
     DATA_PROTECTION = "DATA_PROTECTION"
     GAME_CENTER = "GAME_CENTER"
     HEALTHKIT = "HEALTHKIT"
     HOMEKIT = "HOMEKIT"
     HOT_SPOT = "HOT_SPOT"
     ICLOUD = "ICLOUD"
-    IN_APP_PURCHASE = "IN_APP_PURCHASE"
     INTER_APP_AUDIO = "INTER_APP_AUDIO"
+    IN_APP_PURCHASE = "IN_APP_PURCHASE"
     MAPS = "MAPS"
     MULTIPATH = "MULTIPATH"
+    NETWORK_CUSTOM_PROTOCOL = "NETWORK_CUSTOM_PROTOCOL"
     NETWORK_EXTENSIONS = "NETWORK_EXTENSIONS"
+    NETWORK_SLICING = "NETWORK_SLICING"
     NFC_TAG_READING = "NFC_TAG_READING"
     PERSONAL_VPN = "PERSONAL_VPN"
     PUSH_NOTIFICATIONS = "PUSH_NOTIFICATIONS"
     SIRIKIT = "SIRIKIT"
+    SYSTEM_EXTENSION_INSTALL = "SYSTEM_EXTENSION_INSTALL"
+    USER_MANAGEMENT = "USER_MANAGEMENT"
     WALLET = "WALLET"
     WIRELESS_ACCESSORY_CONFIGURATION = "WIRELESS_ACCESSORY_CONFIGURATION"
 
+    @classmethod
+    def from_display_name(cls, display_name: str) -> CapabilityType:
+        for capability_type in cls:
+            if capability_type.display_name == display_name:
+                return capability_type
+        raise ValueError("Unknown capability type", display_name)
+
+    @property
+    def display_name(self) -> str:
+        try:
+            return {
+                CapabilityType.ACCESS_WIFI_INFORMATION: "Access Wi-Fi Information",
```

Comment:
```suggestion
                self.ACCESS_WIFI_INFORMATION: "Access Wi-Fi Information",
```

Reply:
This is not compliant with mypy type checks. Skipping suggestion.